app/controllers/admin/reservations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Admin::ReservationsController < ApplicationController
```

Comment:
Makes sense.. We had discussed doing requested ect.. but I think this makes sense too.